ci/pipeline.yaml

```diff
@@ -153,16 +142,13 @@ jobs:
           trigger: true
           passed:
             - test
-        - get: version
-          params:
-            bump: patch
       - in_parallel:
         - do:
           - task: build
             image: test-image
             file: source/ci/tasks/build.yaml
           - put: release
```

Comment:
Shouldn't the release also be removed now? Otherwise we are going to release twice?

Reply:
Done by @petewall